libs/build_dod_dataset.py

```diff
@@ -158,26 +184,28 @@ def get_county_projections():
         path = os.path.join(OUTPUT_DIR_COUNTIES, file_name)
         # if the file exists in that directory then process
         if os.path.exists(path):
-            with open(path, "r") as projections:
```

Comment:
i wonder if my `fips = fips_row['fips']` trims the leading zerro

Reply:
pretty unrelated but should try to save our fips codes in csvs surrounded by quotes so we don't always have to fill em in